packages/gestalt/src/Video.js

```diff
@@ -549,7 +552,10 @@ export default class Video extends PureComponent<Props, State> {
             onSeeked={this.handleSeek}
             onTimeUpdate={this.handleTimeUpdate}
             onProgress={this.handleProgress}
-            {...(crossOrigin ? { crossOrigin } : null)}
+            {...(objectFit ? { style: { 'object-fit': objectFit } } : null)}
+            {...((crossOrigin ? { crossOrigin } : { ...null }): {|
```

Comment:
@AlbertCarreras / @jennyscript fyi - a pattern we might see more often when we spread on a component. It's pretty verbose but better than adding a `$FlowIssue` / `$FlowFixMe`: https://github.com/facebook/flow/issues/8186#issuecomment-555747680

The `{ ...null }` seems odd but it's what we have to do to make it work with exact types